dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCSameContextRootTest.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
```

Comment:
What was your rationale for adding this test to the Servlet 4.0 bucket?

Reply:
I usually put the test in the version/release where the fix code has been.  There is no particular reason for this case because the fix is at the base which applies to all releases.  I can move it to FAT 6.0 if needed.